tests/functional/strategy/test_attacks.py

```diff
@@ -0,0 +1,55 @@
+from brownie import Wei, reverts
+import brownie
+
+
+def test_sandwich_attack(TestStrategy, web3, token, gov, vault, strategist, rando):
+
+    honest_lp = gov
+    attacker = rando
+    balance = token.balanceOf(honest_lp) / 2
+
+    # seed attacker their funds. Same amount as deposited in vault
+    token.transfer(attacker, balance, {"from": honest_lp})
+    bv = print(token.balanceOf(vault) / 1e18)
+    vault.withdraw(vault.balanceOf(honest_lp) / 2, {"from": honest_lp})
+    bv = print(token.balanceOf(vault) / 1e18)
+
+    # our actors:
+    # attacker is attacker
+    # strategist is vault owner
+    # honest_lp is a normal lp
+
+    # ------ setup vault ------
+    # we don't use the one in conf because we want no rate limit
+    strategy = strategist.deploy(TestStrategy, vault)
+    vault.addStrategy(
+        strategy, token.totalSupply(), token.totalSupply(), 50, {"from": gov}
+    )
```

Comment:
Just change the rate limit to `2 ** 256 -1` to disable it instead

Reply:
fixed